README.md

```diff
@@ -166,6 +166,32 @@ receivers:
   - service_key: <team-DB-key>
 ```
 
+## High Availability
+
+> Warning: High Availablility is under active development
+
+To create a highly available cluster of the alertmanager the instances need to
+be started in a mesh network. To configure the mesh use the following
+parameters:
```

Comment:
Mesh network might be a bit too abstract. Maybe describe more freely like "... the Alertmanager instances need to be configured to communicate with each other. This is configured via the `-mesh.*` flags.

Clarify that nickname and hardware address do not have to be explicitly set. Clarify that the port used in `mesh.listen-address` is the one to be used in `mesh.peer`. We should say that `mesh.peer` has to point to at least one living Alertmanager at the time of startup.